### PR TITLE
change logs to the new method that naaman wrote

### PR DESCRIPTION
**Pull request by ItayBenMoshe**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx [hunk 1]
```diff
@@ -53,18 +53,12 @@ const ContactUploader = ({ contactEvent, onSave, allInteractions }: ExcelUploade
 
     const saveDataInFile = (contacts?: ParsedExcelRow[]) => {
         if(contacts && contacts.length > 0) {
-            const logInfo = {
-                service: Service.CLIENT,
+            const DataInFileLogger = logger.setup({
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx:56]: Change from pascal case to canal case

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 2]
```diff
@@ -34,80 +32,46 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
...
         if (!checkDuplicateIds(allContactedInteractions.map((contact: InteractedContact) => contact.identificationNumber))) {
             const contactsSavingVariable = {
                 unSavedContacts: {contacts: allContactedInteractions}
             }
-            logger.info({
-                service: Service.CLIENT,
-                severity: Severity.LOW,
-                workflow,
-                step: `launching server request with parameter: ${JSON.stringify(contactsSavingVariable)}`,
+            const ContactLogger = logger.setup({
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:61]: Change from pascal case to camal case

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 3]
```diff
@@ -131,155 +95,77 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
     }
 
     const loadInteractedContacts = () => {
-        logger.info({
-            service: Service.CLIENT,
-            severity: Severity.LOW,
+        const InteractedContactsLogger = logger.setup({
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:98]: Pascal to camal case

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 4]
```diff
@@ -131,155 +95,77 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
...
-            })
+            InteractedContactsLogger.error(`got the following error from the server: ${err}`, Severity.LOW);
         })
     }
 
     const loadFamilyRelationships = () => {
-        logger.info({
-            service: Service.CLIENT,
-            severity: Severity.LOW,
+        const FamilyRelationshipsLogger = logger.setup({
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:121]: Pascal to camal case

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 5]
```diff
@@ -131,155 +95,77 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
...
-            });
+            FamilyRelationshipsLogger.error(`got the following error from the server: ${err}`, Severity.LOW);
         });
     }
 
     const loadContactStatuses = () => {
-        logger.info({
-            service: Service.CLIENT,
-            severity: Severity.LOW,
+        const ContactStatusesLogger = logger.setup({
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:140]: Pascal to camal case

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx (hunk 1 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 2 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 3 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 4 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 5 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

**Review by NaamanTsur — APPROVED** [review]

**Review by nevobonneh — APPROVED** [review]

**Pull request merged**